SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -222,6 +222,9 @@ namespace SpatialConstants
 	const FString SPATIALOS_METRICS_DYNAMIC_FPS = TEXT("Dynamic.FPS");
 
 	const FString LOCATOR_HOST = TEXT("locator.improbable.io");
+	const FString RECONNECT_USING_ARGUMENTS = TEXT("0.0.0.0");
```

Comment:
I think some comment here explaining what this is used for and a docs update would be cool for this new functionality.
Also the name could be more specific to `RECONNECT_USING_PREVIOUS_ARGS` or `RECONNECT_USING_COMMANDLINE_ARGS` or something :D

Reply:
Done